pylint/checkers/exceptions.py

```diff
@@ -254,6 +255,15 @@ class ExceptionsChecker(checkers.BaseChecker):
 
     def open(self) -> None:
         self._builtin_exceptions = _builtin_exceptions()
+        if any(
```

Comment:
Shouldn't this be the other way around? As in `not any`?

Either way, it might even be prettier to do `for exc_name in ...` as we can than create a warning with a very specific exception that name that needs to be changed. We could even suggest `builtins.{exc_name}`.

Reply:
Since this is something users can change themselves and they are not dependent on sub-packages fixing their reliance on deprecated functionality I think it makes sense to warn on each infraction.